besu/src/main/java/org/hyperledger/besu/services/BesuPluginContextImpl.java

```diff
@@ -153,6 +170,11 @@ public void stopPlugins() {
     state = Lifecycle.STOPPED;
   }
 
+  @Override
+  public List<String> getPluginVersions() {
```

Comment:
nit: can probably return a collection rather than a List (slightly more generic) [just on the interface, body can stay the same]

Reply:
agree. Regarding your above point about caching plugin versions, it saves calculation here. If you feel strongly about not caching at registration time let me know, I'll move the plugin version calculation here instead.